Dockerfile.template

```diff
@@ -1,17 +1,22 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-node:latest
+FROM resin/%%RESIN_MACHINE_NAME%%-node:slim
 
-## uncomment if you want systemd
-# ENV INITSYSTEM on
+# Install wget and curl
+RUN apt-get clean && apt-get update && apt-get install -y \
+  wget \
+  curl
```

Comment:
The way docker works each layer is built as an overlay the previous layer, which means that once a file is included in a layer that space will always be used even if it is deleted (in reality shadowed) by a later layer.